clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/KubernetesManifestAnnotater.java

```diff
@@ -228,6 +234,21 @@ public static KubernetesCachingProperties getCachingProperties(KubernetesManifes
         .build();
   }
 
+  public static void useSourceCapacity(KubernetesHandler deployer, KubernetesManifest manifest, KubernetesV2Credentials credentials) {
+    Map<String, String> annotations = manifest.getAnnotations();
+    if (deployer instanceof CanScale &&
+        annotations.containsKey(USE_SOURCE_CAPACITY) &&
+        annotations.get(USE_SOURCE_CAPACITY).equals("true")) {
```

Comment:
The logic for grabbing the source capacity arguably shouldn't live in this class -- we only read & set annotations here.

Reply:
I would add `useSourceCapacity` to the `KubernetesManifestStrategy.java` class, and then in a separate class do the lookup of the capacity.

Reply:
makes sense.